test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -380,7 +380,7 @@
             },
             "bundle_uuid": "aaa96233-bf27-44c7-82df-b4dc15ad4d9d",
             "bundle_version": "2018-11-02T113344.698028Z",
-            "bundle_deleted": false
+            "bundle_deleted": 0
```

Comment:
Should we map this field as None so it is not translated? It can never be None.

Reply:
Should this change be gone now that we don't translate this field anymore?

Reply:
Hm, bug?
```
from azul.json_freeze import freeze
freeze({'foo': False}) == freeze({'foo': 0})
True
```

Reply:
Looks like it. Could you file a bug?

Reply:
Bug ticket created https://github.com/DataBiosphere/azul/issues/1423